nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/util/DataTypeUtils.java

```diff
@@ -2078,4 +2078,8 @@ public static boolean isFittingNumberType(final Object value, final RecordFieldT
 
         return false;
     }
+
+    private static boolean isBlank(String string) {
+        return string == null || string.isBlank();
```

Comment:
Based on the usage, the `string == null` check appears to be unnecessary. With that, it looks like this method could be removed and `string.isBlank()` could be used directly.